metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -152,23 +152,23 @@ class MetalsLanguageServer(
   val excludedPackageHandler: ExcludedPackagesHandler =
     new ExcludedPackagesHandler(userConfig.excludedPackages)
   var ammonite: Ammonite = _
-  val buildTargets: BuildTargets = BuildTargets.withAmmonite(() => ammonite)
+  val buildTargetz: BuildTargets = BuildTargets.withAmmonite(() => ammonite)
```

Comment:
If you have a better name for this, I'm all ears. It conflicts with the new `buildTargets` method down below so I needed to change it.